### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rfostii**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791650%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791798%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791949%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792018%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792036%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792181%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792486%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792547%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792586%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792688%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7818550%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7820210%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7820288%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7820371%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7820385%22%2C%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7821275%22%2C%20%22https%3A//github.com/nromanen/ch017/commit/f53d457b41ead43ee5d40dfbed8d00896a64ea28%23commitcomment-4555702%22%2C%20%22https%3A//github.com/nromanen/ch017/commit/39a4e21b8a2231a10ef00e4ff8a95a113fb02f38%23commitcomment-4586652%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/controllers/TodoController.js%2054%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791798%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22think%20about%20whether%20we%20need%20it%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A29%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/controllers/TodoController.js%3AL1-102%22%7D%2C%20%22Commit%2039a4e21b8a2231a10ef00e4ff8a95a113fb02f38%20tests/karma.conf.js%201%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/commit/39a4e21b8a2231a10ef00e4ff8a95a113fb02f38%23commitcomment-4586652%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22in%20this%20file%20I%20wrote%20my%20own%20pathes%20to%20files.%20If%20you%20wanna%20set%20your%20own%20pathes%20please%20make%20git%20pull%2C%20add%20this%20file%20to%20.gitignore%20and%20rewrite%20pathes%20using%20your%20own%20pathes.%22%2C%20%22created_at%22%3A%20%222013-11-13T11%3A05%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20tests/karma.conf.js%3AL1%20%2839a4e21%29%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/templates/auth.html%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792547%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22try%20to%20see%20if%20angular%20or%20bootstrap%20can%20do%20this%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A52%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20was%20created%20directives%20for%20login/password%2C%20which%20includes%20the%20same%20patterns.%22%2C%20%22created_at%22%3A%20%222013-11-21T10%3A18%3A00Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/templates/auth.html%3AL1-21%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/project-conf.js%257E%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792486%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22remove%20this%20file%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A50%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/project-conf.js%7E%3AL1-26%22%7D%2C%20%22Commit%20f53d457b41ead43ee5d40dfbed8d00896a64ea28%20js/controllers.js%2052%2040%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/commit/f53d457b41ead43ee5d40dfbed8d00896a64ea28%23commitcomment-4555702%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22sorry%20I%20mean%200%20in%20javascript%20is%20false%22%2C%20%22created_at%22%3A%20%222013-11-09T13%3A26%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20js/controllers.js%3AL40%20%28f53d457%29%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/controllers/AuthController.js%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791650%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20file%20will%20be%20reworked%20totally%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A24%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222013-11-21T11%3A12%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/controllers/AuthController.js%3AL1-82%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/controllers/UserListController.js%2017%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792036%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22remove%20this%20and%20finalize%20the%20controller%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A37%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/controllers/UserListController.js%3AL1-20%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/controllers/TodoController.js%2083%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7791949%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22remove%20this%20from%20this%20method%20and%20methods%20below%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A34%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/controllers/TodoController.js%3AL1-102%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/project-conf.js%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792181%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22patient%20list%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A41%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222013-11-21T10%3A22%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/project-conf.js%3AL1-21%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/templates/auth.html%2011%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792586%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22simply%20%60%60%60required%60%60%60%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A53%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222013-11-21T10%3A14%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/templates/auth.html%3AL1-21%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/templates/auth.html%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792688%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22what%20is%20ng-bind%3F%20do%20we%20really%20need%20it%3F%20can%20we%20use%20ng-model%3F%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A56%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22Simply%20this%20is%20an%20analog%20of%20%7B%7BhintText%7D%7D.%22%2C%20%22created_at%22%3A%20%222013-11-21T09%3A02%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/templates/auth.html%3AL1-21%22%7D%2C%20%22Pull%2042d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab%20frontend/js/controllers/UserListController.js%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/nromanen/ch017/pull/3%23discussion_r7792018%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22user%20rename%20to%20patient%20in%20code%20and%20file%20name.%22%2C%20%22created_at%22%3A%20%222013-11-20T14%3A36%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3430069%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rfostii%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222013-11-21T10%3A23%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5704122%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ArturDolgii%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/js/controllers/UserListController.js%3AL1-20%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Commit f53d457b41ead43ee5d40dfbed8d00896a64ea28 js/controllers.js 52 40'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/commit/f53d457b41ead43ee5d40dfbed8d00896a64ea28#commitcomment-4555702'>File: js/controllers.js:L40 (f53d457)</a></b>
- <a href='https://github.com/rfostii'></a> sorry I mean 0 in javascript is false
- [ ] <a href='#crh-comment-Commit 39a4e21b8a2231a10ef00e4ff8a95a113fb02f38 tests/karma.conf.js 1 1'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/commit/39a4e21b8a2231a10ef00e4ff8a95a113fb02f38#commitcomment-4586652'>File: tests/karma.conf.js:L1 (39a4e21)</a></b>
- <a href='https://github.com/rfostii'></a> in this file I wrote my own pathes to files. If you wanna set your own pathes please make git pull, add this file to .gitignore and rewrite pathes using your own pathes.
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/controllers/AuthController.js 1'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7791650'>File: frontend/js/controllers/AuthController.js:L1-82</a></b>
- <a href='https://github.com/rfostii'></a> this file will be reworked totally
- <a href='https://github.com/ArturDolgii'></a> done
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/controllers/TodoController.js 54'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7791798'>File: frontend/js/controllers/TodoController.js:L1-102</a></b>
- <a href='https://github.com/rfostii'></a> think about whether we need it
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/controllers/TodoController.js 83'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7791949'>File: frontend/js/controllers/TodoController.js:L1-102</a></b>
- <a href='https://github.com/rfostii'></a> remove this from this method and methods below
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/controllers/UserListController.js 1'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792018'>File: frontend/js/controllers/UserListController.js:L1-20</a></b>
- <a href='https://github.com/rfostii'></a> user rename to patient in code and file name.
- <a href='https://github.com/ArturDolgii'></a> done
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/controllers/UserListController.js 17'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792036'>File: frontend/js/controllers/UserListController.js:L1-20</a></b>
- <a href='https://github.com/rfostii'></a> remove this and finalize the controller
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/project-conf.js 10'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792181'>File: frontend/js/project-conf.js:L1-21</a></b>
- <a href='https://github.com/rfostii'></a> patient list
- <a href='https://github.com/ArturDolgii'></a> done
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/js/project-conf.js%7E 1'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792486'>File: frontend/js/project-conf.js~:L1-26</a></b>
- <a href='https://github.com/rfostii'></a> remove this file
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/templates/auth.html 5'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792547'>File: frontend/templates/auth.html:L1-21</a></b>
- <a href='https://github.com/rfostii'></a> try to see if angular or bootstrap can do this
- <a href='https://github.com/ArturDolgii'></a> I was created directives for login/password, which includes the same patterns.
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/templates/auth.html 11'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792586'>File: frontend/templates/auth.html:L1-21</a></b>
- <a href='https://github.com/rfostii'></a> simply `required`
- <a href='https://github.com/ArturDolgii'></a> done
- [ ] <a href='#crh-comment-Pull 42d8c61b80ff66c0a3a15fd07fb27bb17f8f48ab frontend/templates/auth.html 16'></a> &nbsp;<b><a href='https://github.com/nromanen/ch017/pull/3#discussion_r7792688'>File: frontend/templates/auth.html:L1-21</a></b>
- <a href='https://github.com/rfostii'></a> what is ng-bind? do we really need it? can we use ng-model?
- <a href='https://github.com/ArturDolgii'></a> Simply this is an analog of {{hintText}}.

<a href='https://www.codereviewhub.com/nromanen/ch017/pull/3?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/nromanen/ch017/pull/3?approve=1'></a>&nbsp;<a href='https://github.com/nromanen/ch017/pull/3'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: frontend/js/controllers/AuthController.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
```
Comment (rfostii) [inline frontend/js/controllers/AuthController.js:1]: this file will be reworked totally

File: frontend/js/controllers/TodoController.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    };
+
+    $scope.canCheckTodo = function () {
+        return $scope.statusInSystem.rights.check;
+    };
+
+    $scope.changeStatusInSystem = function(role) {
+        $scope.statusInSystem = role;
+
+        return role;
```
Comment (rfostii) [inline frontend/js/controllers/TodoController.js]: think about whether we need it

File: frontend/js/controllers/TodoController.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            }
+        });
+        return count;
+    };
+
+    $scope.clearDoneTodos = function() {
+        $scope.todoList = $scope.todoList.filter(function(todo) {
+            return !todo.done;
+        });
+        updateLocalStorage();
```
Comment (rfostii) [inline frontend/js/controllers/TodoController.js]: remove this from this method and methods below

File: frontend/js/controllers/UserListController.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (rfostii) [inline frontend/js/controllers/UserListController.js]: user rename to patient in code and file name.

File: frontend/js/controllers/UserListController.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        {name: 'John'},
+        {name: 'Andrew'},
+        {name: 'Nick'},
+        {name: 'Andrew'},
+        {name: 'John J'}
+    ];
+
+    $scope.getActivePatient = function () {
+        $scope.currentPatient = this.user.name;
+        console.log($scope.currentPatient)
```
Comment (rfostii) [inline frontend/js/controllers/UserListController.js]: remove this and finalize the controller

File: frontend/js/project-conf.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+var App = angular.module(
+   "App",
+   [
+       "LocalStorageModule"
+   ]
+).config(function ($routeProvider) {
+
+    $routeProvider.when('/', {controller: 'TodoController', templateUrl: 'templates/todo.html'});
+    $routeProvider.when('/auth', {controller: 'AuthController', templateUrl: 'templates/auth.html'});
+    $routeProvider.when('/userList', {controller: 'UserListController', templateUrl: 'templates/userList.html'});
```
Comment (rfostii) [inline frontend/js/project-conf.js]: patient list

File: frontend/js/project-conf.js~ [hunk 7]
```diff
@@ -0,0 +1,25 @@
+var App = angular.module(
```
Comment (rfostii) [inline frontend/js/project-conf.js~]: remove this file

File: frontend/templates/auth.html [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<div ng-controller="AuthController">
+    <form class="form-signin" ng-submit="sendData()">
+        <h2 class="form-signin-heading">Please sign in</h2>
+
+        <input type="text" ng-model="authLogin" pattern="^[a-zA-Z][a-zA-Z0-9-_\.]{2,}$"
```
Comment (rfostii) [inline frontend/templates/auth.html]: try to see if angular or bootstrap can do this

File: frontend/templates/auth.html [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+               title="At least 3 characters" class="form-control" placeholder="Your login"
+               required="" autofocus="">
+
+        <input type="password" ng-model="authPassword" pattern="[^]{4,}$"
+               title="At least 4 characters" class="form-control"
+               placeholder="Password" required="">
```
Comment (rfostii) [inline frontend/templates/auth.html]: simply `required`

File: frontend/templates/auth.html [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+        <label class="checkbox">
+            <input type="checkbox" ng-model="authRemember" value="remember-me"> Remember me
+        </label>
+        <div ng-bind="hintText" ng-style="hintStyle" class="hint"></div>
```
Comment (rfostii) [inline frontend/templates/auth.html]: what is ng-bind? do we really need it? can we use ng-model?

Comment (ArturDolgii) [inline frontend/templates/auth.html]: Simply this is an analog of {{hintText}}.

File: frontend/templates/auth.html (hunk 9 above)
Comment (ArturDolgii) [inline frontend/templates/auth.html]: done

File: frontend/templates/auth.html (hunk 8 above)
Comment (ArturDolgii) [inline frontend/templates/auth.html]: I was created directives for login/password, which includes the same patterns.

File: frontend/js/project-conf.js (hunk 6 above)
Comment (ArturDolgii) [inline frontend/js/project-conf.js]: done

File: frontend/js/controllers/UserListController.js (hunk 4 above)
Comment (ArturDolgii) [inline frontend/js/controllers/UserListController.js]: done

File: frontend/js/controllers/AuthController.js (hunk 1 above)
Comment (ArturDolgii) [inline frontend/js/controllers/AuthController.js:1]: done

**Pull request closed without merging**